### DISCUSSION TITLE
It would be nice if pinned requests, sticky requests area can be extended with doubled headed arrow vertically

### DISCUSSION THREAD

**Discussion by aungthanlwin** [Kong/insomnia#5232] [Ideas]
When there are too many requests pinned, I have to scroll up and down to look for the particular request. If I can extend and adjust area vertically, I can see all my pinned requests and that would be nice feature addition. (We already able to extend the pinned area horizontally)

![image](https://user-images.githubusercontent.com/13598164/192857262-7002c8e4-2893-4414-b046-f15781b3f36b.png)

**Comment by adityahas** [comment]
Completely agree, the pinned area pane should be resizable.
